driver/InputFlags.cpp

```diff
@@ -124,9 +127,30 @@ void InputFlags::Parse(int argc, char* argv[])
                 Print();
 
             char short_name = FindShortName(long_name);
-
-            MapInputs[short_name].value = args[i + 1];
-            i++;
+            
+            //check for short name of in layout flags
+            if ( (short_name == 'I') || (short_name == 'O') || (short_name == 'f'))
+            {
+              //check layout flag values
+              if ((args[i + 1]) == ("NCHW")||
```

Comment:
The driver should not limit layouts. Only library knows which layouts are supported.